plugins/modules/identity/ipa/ipa_otpconfig.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021, Ansible Project
+# Heavily influenced from Fran Fitzpatrick <francis.x.fitzpatrick@gmail.com> ipa_config module
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: ipa_otpconfig
+author: justchris1 (@justchris1)
+short_description: Manage FreeIPA OTP Configuration Settings
+version_added: 2.5.0
+description:
+- Modify global configuration settings of a FreeIPA Server with respect to OTP (One Time Passwords).
+options:
+  ipatokentotpauthwindow:
+    description: TOTP authentication window in seconds.
+    aliases: ["totpauthwindow"]
+    type: int
+  ipatokentotpsyncwindow:
+    description: TOTP synchronization window. (Seconds)
+    aliases: ["totpsyncwindow"]
+    type: int
+  ipatokenhotpauthwindow:
+    description: HOTP authentication window in number of hops.
+    aliases: ["hotpauthwindow"]
+    type: int
+  ipatokenhotpsyncwindow:
+    description: HOTP synchronization window. (Hops)
+    aliases: ["hotpsyncwindow"]
+    type: int
+extends_documentation_fragment:
+- community.general.ipa.documentation
+
+'''
+
+EXAMPLES = r'''
+- name: Ensure the TOTP authentication window is set to 300 seconds
+  community.general.ipa_otpconfig:
+    ipatokentotpauthwindow: '300'
+    ipa_host: localhost
+    ipa_user: admin
+    ipa_pass: supersecret
+
+- name: Ensure the TOTP syncronization window is set to 86400 seconds
+  community.general.ipa_otpconfig:
+    ipatokentotpsyncwindow: '86400'
+    ipa_host: localhost
+    ipa_user: admin
+    ipa_pass: supersecret
+
+- name: Ensure the HOTP authentication window is set to 10 hops
+  community.general.ipa_otpconfig:
+    ipatokenhotpauthwindow: '10'
+    ipa_host: localhost
+    ipa_user: admin
+    ipa_pass: supersecret
+
+- name: Ensure the HOTP syncronization window is set to 100 hops
+  community.general.ipa_otpconfig:
+    ipatokenhotpsyncwindow: '100'
+    ipa_host: localhost
+    ipa_user: admin
+    ipa_pass: supersecret
+'''
+
+RETURN = r'''
+otpconfig:
```

Comment:
The module returns something called `users`, but not something called `optconfig`.

Reply:
Resolved in 59f47ff